sdk/ledger-test-tool/tool/BUILD.bazel

```diff
@@ -183,52 +183,6 @@ conformance_test_excluded_test = [
     "ActiveContractsServiceIT:AcsAtPruningOffsetIsAllowed",  # requires pruning not available in canton community
 ]
 
-# conformance tests for default and latest
-#  those test the conformance test it-self for stable features
-conformance_test(
-    name = "conformance-test",
-    extra_data = conformance_test_extra_data_base + ["//canton:community_app_deploy.jar"],
-    extra_runner_args = ["7000"],
-    lf_versions = [
-        "default",
-        "latest",
-    ],
-    ports = conformance_test_ports,
-    runner = "@//bazel_tools/client_server/runner_with_health_check",
-    server = ":canton-test-runner-with-dependencies",
-    server_args = conformance_server_args_base + [
-        "-C",
-        ",".join([
-            "dev-mode=no",
-            "_shared.participant_ledger-api_tls.minimum-server-protocol-version=TLSv1.3",
-        ]),
-    ],
-    tags = ["main-only"],
-    test_tool_args = conformance_test_tool_args_base + [
-        "--exclude=" + ",".join(conformance_test_excluded_test),
-        "--additional=TLSOnePointThreeIT",
-    ],
-) if not is_windows and is_intel else None
-
-# this tests TLSAtLeastOnePointTwoIT
-conformance_test(
-    name = "conformance-test-tls1.2-or-newer",
-    extra_data = conformance_test_extra_data_base + ["//canton:community_app_deploy.jar"],
-    extra_runner_args = ["7000"],
-    lf_versions = ["latest"],
-    ports = conformance_test_ports,
-    runner = "@//bazel_tools/client_server/runner_with_health_check",
-    server = ":canton-test-runner-with-dependencies",
-    server_args = conformance_server_args_base + [
-        "-C",
-        ",".join([
-            "dev-mode=no",
-        ]),
-    ],
-    tags = ["main-only"],
-    test_tool_args = conformance_test_tool_args_base + ["--include=TLSAtLeastOnePointTwoIT"],
-) if not is_windows and is_intel else None
-
```

Comment:
I will take a look at it on thursday, so it is not so important anyway.

Reply:
Just a guess but I think this might be related to https://github.com/DACH-NY/canton/pull/18662, since the other two conformance tests in this file, which do pass, set the version explicitly as `dev`

```bazel
        server_args = conformance_server_args_base + [
            "-C",
            ",".join([
                "dev-mode=yes",
                "canton.domains.test_domain.init.domain-parameters.protocol-version=dev",
            ]),
        ],
```

Reply:
I opened (draft) https://github.com/digital-asset/daml/pull/19122 to test this theory